db/migrations/.eslintrc

```diff
@@ -0,0 +1,3 @@
+{
+  extends: "eslint-config-lob/migrations"
```

Comment:
This can be [shortened down](http://eslint.org/docs/user-guide/configuring#extending-configuration-files) to `extends: "lob/migrations"`.

> You can omit eslint-config- and ESLint will automatically insert it for you, similar to how plugins work. See Shareable Configs for more information.